Enable monitoring in sysgo (#18017)
ethereum-optimism/optimism: op-devstack/README.md

- `DEVSTACK_L1EL_KIND=geth` to select geth as default L1 EL node
- `SYSGO_GETH_EXEC_PATH=/path/to/geth` to select the geth executable to run

L2 Observability env vars (note: only applicable if any L2 component calls `Orchestrator.RegisterL2MetricsEndpoints`):

> - `SYSGO_DOCKER_EXEC_PATH` to set local path to docker executable (defaults to `/usr/local/bin/docker`)

Reviewer: Nit: can we just assume that docker is in the path ENV var and remove that env variable? In general that's the case

Author: How do you feel about
```suggestion
- `SYSGO_DOCKER_EXEC_PATH` path to docker executable (defaults to `docker` assuming it is in `PATH`)
```

Author: Updated here: https://github.com/ethereum-optimism/optimism/pull/18017/commits/c61c6126d8c854433c85ee44e74f407d0261ef0e

Reviewer: That works too!
